pkg/storage/internal/database/models/output.go

```diff
@@ -0,0 +1,31 @@
+package models
+
+import (
+	"gorm.io/gorm"
+)
+
+type Output struct {
+	gorm.Model
+
+	UserID        int    `gorm:"index"`
+	TransactionID uint   `gorm:"index"`
+	Vout          uint32 `gorm:"index"`
+	Satoshis      int64
+
+	LockingScript      *string
+	CustomInstructions *string
```

Comment:
Just a thought:
for `CustomInstructions` or `Description` to maybe put a `gorm:"type:text"` if we expect those fields to contain big strings.
But I am not sure if we want to support something more than sqlite for now.